Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
```

Comment:
Not related to changes, but this PR seems like a sensible place to include this change:

We can replace:
```rust
tokio = { version = "1", features = ["full"] }
```
with:
```rust
tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
```

to reduce build time a bit, as we are only currently using macros (and the default runtime, `rt-multi-thread`)

Reply:
Sounds good, changed it!